test_files/fields.in.ts

```diff
@@ -7,7 +7,11 @@ class Klass {
   }
 
   getF1() {
-    // This access print a warning without a generated field stub declaration.
+    // This access prints a warning without a generated field stub declaration.
     return this.field1;
   }
 }
+
+let fieldsTest = new FieldsTest(3);
+// Ensure the type is understood by Closure.
+fieldsTest.field1 = 'hi';
```

Comment:
This right here fails to compile if Closure doesn't believe that "field1" is a string, so by including it in the test here we get coverage of the Closure check that the field exists.